packages/cron/src/lib/cid.js

```diff
@@ -0,0 +1,24 @@
+import { CID } from 'multiformats/cid'
+import { InvalidCidError } from './errors.js'
+
+/**
+ * Parse CID and return normalized b32 v1
+ *
+ * @param {string} cid
+ */
+export function normalizeCid (cid) {
+  try {
+    const c = CID.parse(cid)
+    return c.toV1().toString()
+  } catch (err) {
+    throw new InvalidCidError(cid)
+  }
+}
+
+export function denormalizeCid (cid) {
```

Comment:
It's denormalized in the sense that it's an encoding that's not the normalized encoding...but I think "downgrade" is a better description of what's happening here.

```suggestion
export function downgradeCid (cid) {
```